.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,19 +2,17 @@
 name: ✨ Feature request
 about: Suggest an idea for this project
 ---
+## Feature request
 <!--- Verify first that your feature was not already discussed on GitHub -->
 <!--- Complete *all* sections as described, this form is processed automatically -->
 
-##### SUMMARY
+### Summary
 <!--- Describe the new feature/improvement briefly below -->
 
-##### ISSUE TYPE
-- Feature Idea
```

Comment:
Did this mean to be removed, this is how the `feature` label gets added

Reply:
Yes, ansibot needs to understand the new format too.